portfolio/src/main/java/com/google/sps/servlets/models/PubSubHandler.java

```diff
@@ -17,54 +17,38 @@
 import java.io.IOException;
 import java.lang.InterruptedException;
 
-public class PublishHandler {
+public class PubSubHandler {
```

Comment:
Rename this something like PubSubUtils now that it's no longer actually handling anything.  Handler is an object.  This is not an object it is a set of static methods.